File: ExplicitTypeTest/results/ExplicitTypeTest.cpp
#include <one.hpp>
#include <iostream>

class TestClass {
  public:
    void testMethod() {
        auto op = nullptr;
        std::cout << op.size() << std::endl;
    }

  private:
};

int main()
{
    try {
        TestClass c;
        c.testMethod();
    } catch(std::exception& err) {
        std::cout << "Exception: " << err.what() << '\n';
    }
    return 0;
}

File: CompilationTest/ExplicitTypeTest/results/ExplicitTypeTest.cs
using System;

public class TestClass
{
    public void TestMethod()
    {
        var op = "x";
        Console.WriteLine(op.Length);
    }
}

public class Program
{
    static public void Main(string[] args)
    {
        try 
        {
            new TestClass().TestMethod();
        }
        catch (System.Exception e)
        {
            System.Console.WriteLine($"Exception: {e.Message}");
        }
    }
}